conf/config.env

```diff
@@ -0,0 +1,8 @@
+# This file is not meant to be edited. It is meant to be configured through the config panel.
```

Comment:
Wonder if we should not rather put the env variable in `custom.env` instead.

I wonder what happens if the env variables are missing. Maybe the config panel works fine anyway?